### DISCUSSION TITLE
Coloring every other added row

### DISCUSSION THREAD

**Discussion by vaskoZGU** [Budibase/budibase#11869] [Help] [answered]
Hello,

Is it possible to format table coloring every other row as in the screenshot below?

![image](https://github.com/Budibase/budibase/assets/117993754/00530f13-b13c-4597-ab64-2d003ee7dac4)

Thank you

**Comment by ConorWebb96** [comment, marked as answer]
Hey @vaskoZGU,

Are you asking from a table or grid component perspective? I believe theres a demo out there on how to do something similar with the `super table` custom component. You can view this [here](https://github.com/Budibase/budibase/discussions/10204). 

I hope this helps.

**Reply by vaskoZGU** [reply]
Thank you. I'll try with super table.
